src/main/webapp/permissions-manage-files.xhtml

```diff
@@ -25,6 +25,11 @@
             <ui:param name="showMessagePanel" value="false"/>
             <ui:define name="body">
                 <h:form id="rolesPermissionsForm" styleClass="margin-top">
+                    <div class="panel-group">
+                    <p:selectBooleanCheckbox value="#{manageFilePermissionsPage.showDeleted}" itemLabel="#{bundle['dataverse.permissionsFiles.files.includeDeleted']}" onchange="#{manageFilePermissionsPage.init()}" >
```

Comment:
This checkbox for "Include Deleted Files" displays at the top of page, rather than at the top of the table it relates to. I suggest moving it down, next to the file count, above the file table in the Restricted Files panel.

Reply:
I think we need to leave it where it ism because including deleted files also effects the role assignments table and if the check box is hidden in the restricted files panel it will be cumbersome to update the role assignments table.

Reply:
In that case I would suggest that each table gets its own checkbox, rather than one checkbox, which I would say is just as hidden away at the top of the page. If we did mockups before development, the checkboxes would be directly above each of the restricted file tables, next to the file count.

Reply:
@mheppler To reiterate what I said in Slack, it would be good if the two of you sat down together to discuss this and other feedback regarding this issue.

Reply:
@djbrooke Right. I sent him a Slack msg offering to discuss.

Reply:
@mheppler perfect, thanks

Reply:
Thank you.